Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1217,15 +1221,15 @@ ExceptionOr<void> HTMLElement::hidePopoverInternal(FocusPreviousElement focusPre
         queuePopoverToggleEventTask(ToggleState::Open, ToggleState::Closed);
 
     if (RefPtr element = popoverData()->previouslyFocusedElement()) {
-        if (focusPreviousElement == FocusPreviousElement::Yes && containsIncludingShadowDOM(document().focusedElement())) {
+        if (focusPreviousElement == FocusPreviousElement::Yes && containsIncludingShadowDOM(protectedDocument()->focusedElement())) {
```

Comment:
the document is used later on in this function so I would cache it in a local variable right before the `if (RefPtr element = popoverData()->previouslyFocusedElement()) {` check.